web/src/components/FOI/Home/Redlining.js

```diff
@@ -2507,6 +2570,15 @@ const Redlining = React.forwardRef(
             '<?xml version="1.0" encoding="UTF-8" ?><xfdf xmlns="http://ns.adobe.com/xfdf/" xml:space="preserve"><annots>' +
             formattedAnnotationXML +
             "</annots></xfdf>";
+          

```

Comment:
This code is where I got stuck with creating the same redline package as redline, but applying s. 14 redactions. I left the code in here for reference but it can be removed.

I get an error when I try and call stitchObject.applyRedactions(), but am able to applyRedactions on the current document, even though stitchObject and doc are both instances of Document. Not sure why I'm not able to do stitchObject.applyRedactions() but that's where I'm stuck